### PR TITLE
Changes to CLAUDE.md and .claude/

### PR DESCRIPTION
**Pull request by relh**
Read a bunch of tweets about improving claude code. Added a few hooks/commands/etc as examples for people who might want to use them.

[Asana Task](https://app.asana.com/1/1209016784099267/project/1210348820405981/task/1210908302472459)

### CODE REVIEW

File: .claude/hooks/TrainTest [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Get current date and timestamp
+DATE=$(date +%Y%m%d_%H%M%S)
+
+# Run the training command
+echo "Starting training run: cc.test.$DATE"
+uv run ./tools/train.py +hardware=macbook run=cc.test.$DATE trainer.total_timesteps=10000 wandb=off
```
Comment (nishu-builder) [inline .claude/hooks/TrainTest:8]: cc.$USER? even if wandb is off it will eventually still go to policy evalutaor so probably want the identifier

File: CLAUDE.md [hunk 2]
```diff
@@ -28,16 +139,52 @@ metta install aws wandb              # Install specific components
...
+
+1. **Training**: `tools/train.py` - Main training script using Hydra configuration
+   ```bash
+   uv run ./tools/train.py run=my_experiment +hardware=macbook
+   ```
 
-@.cursor/commands.md
+2. **Simulation/Evaluation**: `tools/sim.py` - Run evaluation suites on trained policies
+   ```bash
+   uv run ./tools/sim.py run=eval policy_uri=file://./checkpoints/policy.pt
```
Comment (nishu-builder) [inline CLAUDE.md:153]: we should give an example wandb uri because the format is not obvious

File: CLAUDE.md [hunk 3]
```diff
@@ -28,16 +139,52 @@ metta install aws wandb              # Install specific components
...
+   ```
+
+4. **Interactive Play**: `tools/play.py` - Manual testing and exploration
+   ```bash
+   uv run ./tools/play.py run=play +hardware=macbook
+   ```
+
+5. **Sweep Management**: `tools/sweep_setup.py`, `tools/sweep_prepare_run.py` - Hyperparameter sweep tools
+
+#### Visualization Tools
```
Comment (nishu-builder) [inline CLAUDE.md:168]: until we add puppeteer or something i wonder if this isnt great. when claude code runs some commands like this that hang it takes uncomfortably long to realize it's hanging

File: CLAUDE.md [hunk 4]
```diff
@@ -57,56 +204,72 @@ Not needed, just run scripts, they'll work automatically through uv-powered sheb
...
-- Tests should be focused on testing one thing
-- Tests should cover edge cases and boundary conditions
-- Tests are organized in the `tests/` directory, mirroring the project structure
+#### Debugging Training Issues
 
-## Code Style Guidelines
+1. Enable debug logging: `HYDRA_FULL_ERROR=1`
+2. Use smaller batch sizes for debugging
+3. Check wandb logs for metrics anomalies
+4. Use `tools/play.py` for interactive debugging
```
Comment (nishu-builder) [inline CLAUDE.md:259]: I think cc is unlikely to get good use out of tools/play.py (maybe until we get puppetteer in or something) itself but I could be wrong
